human_parsing example from dnn Assertion failed (int)_numAxes == inputs[0].size()

##### System information (version)

- OpenCV => 4.6
- Operating System / Platform => Windows 64 Bit
- Compiler => Visual Studio 2019

##### Detailed description

human_parsing example from dnn module not working properly no matter of input it every time asserting at:

`modules\dnn\src\layers\permute_layer.cpp:161: error: (-215:Assertion failed) (int)_numAxes == inputs[0].size() in function 'cv::dnn::PermuteLayerImpl::getMemoryShapes'`

I tried to resize / convert input image in 384x384 dimensions and different formats but it still not helping,
I assume the problem is in lip_jppnet_384.pb that is told to use in comments to the example.

##### Steps to reproduce

1. download lip_jppnet_384.pb by [link ](https://www.dropbox.com/s/qag9vzambhhkvxr/lip_jppnet_384.pb?dl=0
) in the example 
2. run [c++](https://github.com/opencv/opencv/blob/4.x/samples/dnn/human_parsing.cpp) or [python example](https://github.com/opencv/opencv/blob/4.x/samples/dnn/human_parsing.py) of human_parsing from dnn module
3. observe assert in output window

##### Issue submission checklist

 - [*] I report the issue, it's not a question
 
 - [*] I checked the problem with documentation, FAQ, open issues,
       forum.opencv.org, Stack Overflow, etc and have not found any solution
  
 - [*] I updated to the latest OpenCV version and the issue is still there
 
 - [*] There is reproducer code and related data files: videos, images, onnx, etc

--- a/modules/dnn/src/tensorflow/tf_importer.cpp
+++ b/modules/dnn/src/tensorflow/tf_importer.cpp
@@ -2541,6 +2541,8 @@ void TFImporter::parsePack(tensorflow::GraphDef& net, const tensorflow::NodeDef&
     if (dim != 0)
         CV_Error(Error::StsNotImplemented, "Unsupported mode of pack operation.");
 
+    data_layouts[name] = DATA_LAYOUT_UNKNOWN;
+
     CV_Assert(hasLayerAttr(layer, "N"));
     int num = (int)getLayerAttr(layer, "N").i();
     CV_CheckEQ(num_inputs, num, "");

